src/main/java/stringcalculator/domain/Separator.java

```diff
@@ -0,0 +1,42 @@
+package stringcalculator.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Separator {
+
+    private final List<String> DELIMITER = Arrays.asList("\\:", "\\,");
+    private final String JOIN_STR = "|";
+    private String customDelimiter = "";
+    private String userNumber = "";
+    private String delimiter = "";
```

Comment:
```suggestion
    private String userNumber;
    private String delimiter;
```

`customDelimiter`의 경우에는 굳이 상태로 둘 필요 없이, `createDelimiterRegex` 메서드(분리한다, 추출한다의 의미가 더 명확하지 않을까요?🤔) 내부 변수로도 충분할 것 같아요. 🤗

Reply:
넵 말씀해주신 부분 반영했습니다 
감사합니다 :)